rollup/internal/controller/watcher/batch_proposer.go

```diff
@@ -149,6 +149,10 @@ func NewBatchProposer(ctx context.Context, cfg *config.BatchProposerConfig, chai
 	return p
 }
 
+func (p *BatchProposer) BatchORM() *orm.Batch {
```

Comment:
as our layout purpose, the orm instance should be exposed to app layer.

Reply:
done in 5ff6fd079976cb1baa9790df5277a947c37f77f7